bindings/pyroot/pythonizations/python/ROOT/_rootcli.py

```diff
@@ -0,0 +1,26 @@
+import os
+import subprocess
+import sys
+
+ROOT_HOME = os.path.dirname(__file__)
+
+
+def main() -> None:
+    # Ensure all ROOT libraries are found at runtime by the process
+    os.environ['LD_LIBRARY_PATH'] = os.path.join(ROOT_HOME, 'lib')
```

Comment:
```suggestion
```
Is this really needed? You build with `rpath=ON`, doesn't the binary contain the relative rpath to the library directory then?